### PR TITLE
WIP: FCREPO-2617. Initial implementation of the Memento creation.

### PR DESCRIPTION
**Pull request by mohideen**
Caveats:
- Partial implementation
- Tests are failing

https://jira.duraspace.org/browse/FCREPO-2617

### CODE REVIEW

**Comment by mohideen** [comment]
This is a work in progress and should not be merged yet. Any feedback would be appreciated.

As there are other issues dependent on this, I have pushed the work I have done so far to the above branch so that someone else can work on this, if necessary. I will try to continue on this next weekend if this is still incomplete. 

@awoods, @whikloj

File: fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java [hunk 1]
```diff
@@ -147,6 +152,8 @@
         Splitter.on(',').omitEmptyStrings().trimResults().
         withKeyValueSeparator(Splitter.on('=').limit(2));
 
+    private static final String RFC_1123_DATE_TIME = "EEE, dd MMM yyyy HH:mm:ss z";
```
Comment (whikloj) [inline fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java]: Can probably reuse https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#RFC_1123_DATE_TIME

**Comment by dbernstein** [comment]
As a note I am currently working on moving this ticket (fcrepo-2617 forward) based on this work.

**Comment by bseeger** [comment]
Can we close this PR now?

**Comment by awoods** [comment]
Resolved with: https://github.com/fcrepo4/fcrepo4/commit/9630a378158d64459788fa469c17544f71c5c620

**Pull request closed without merging**